app/src/main/res/drawable/ic_virus1.xml

```diff
@@ -0,0 +1,9 @@
+    <path android:fillColor="#933" android:pathData="M3.336,97.693a3,3 0,0 1,-2.938 -2.411c-1.31,-6.539 0.619,-12.081 5.735,-16.475a24.129,24.129 0,0 1,15.495 -6.118,4.868 4.868,0 0,0 1.53,-0.217c0.548,-4.076 2.215,-7.56 3.829,-10.931 0.732,-1.53 1.436,-3 2.031,-4.5a27.375,27.375 0,0 1,-7.692 -14.651c-0.961,-5.642 0.55,-10.9 4.146,-14.422s8.824,-4.914 14.383,-3.853c4.179,0.8 9,2.132 13.117,5.954a3.855,3.855 0,0 0,1.132 0.812,4.928 4.928,0 0,0 1.193,-1.545 3,3 0,1 1,5.195 3c-0.577,1 -2.33,4.028 -5.5,4.5 -2.02,0.3 -4.077,-0.5 -6.1,-2.379 -2.991,-2.78 -6.65,-3.784 -10.159,-4.455 -3.651,-0.7 -6.867,0.1 -9.057,2.244s-3.066,5.4 -2.43,9.129a22.088,22.088 0,0 0,7.284 12.645,3 3,0 0,1 0.826,3.143 57.062,57.062 0,0 1,-2.952 6.964c-1.5,3.135 -2.919,6.1 -3.314,9.294 -0.191,1.537 -1.285,5.127 -7.324,5.261a18.288,18.288 0,0 0,-11.719 4.671c-3.489,3 -4.649,6.311 -3.761,10.745a3,3 0,0 1,-2.945 3.59Z"/>
+    <path android:fillColor="#933" android:pathData="M84.787,90.657c-7.336,0 -14.743,-6.613 -15.356,-13.806 -0.744,-8.715 3.243,-14.34 12.188,-17.2a2.993,2.993 0,0 1,1.168 -0.132c0.092,-0.008 0.182,-0.01 0.279,-0.01 5.525,0.057 9.761,1.586 12.59,4.544 2.811,2.94 4.141,7.191 3.953,12.637 -0.235,6.843 -7.406,13.757 -14.478,13.958C85.017,90.655 84.902,90.657 84.787,90.657ZM83.012,65.513c-5.955,2.009 -8.094,5.077 -7.6,10.828 0.348,4.081 5.309,8.46 9.552,8.313 3.915,-0.111 8.525,-4.461 8.652,-8.166 0.13,-3.77 -0.642,-6.557 -2.293,-8.284C89.651,66.459 86.861,65.554 83.012,65.513Z"/>
+    <path android:fillColor="#933" android:pathData="M69.168,116.919q-0.432,0 -0.879,-0.024a11.373,11.373 0,0 1,-10.856 -12.293c0.332,-6.714 5.81,-9.6 7.61,-10.552a2.964,2.964 0,0 1,1.652 -0.336c0.082,-0.007 0.166,-0.011 0.25,-0.011h0.784c0.584,0 1.169,0 1.753,0a12.567,12.567 0,0 1,9.1 3.935,10.393 10.393,0 0,1 2.846,7.809C81.053,111.072 77.056,116.919 69.168,116.919ZM67.214,99.703c-2.572,1.483 -3.68,3.021 -3.787,5.2a5.381,5.381 0,0 0,5.181 6.006c6.126,0.331 6.768,-4.824 6.837,-5.859a4.427,4.427 0,0 0,-1.258 -3.324,6.628 6.628,0 0,0 -4.751,-2.017c-0.566,0 -1.13,0 -1.695,0Z"/>
+</vector>
```

Comment:
Why filename is appended by 1?

Reply:
As I was creating new icons and was trying out that it should not look bad so kept the previous icon too, Though removing now